recipes/so5extra/all/conanfile.py

```diff
@@ -9,15 +9,18 @@ class So5extraConan(ConanFile):
     url = "https://github.com/conan-io/conan-center-index"
     description = "A collection of various SObjectizer's extensions."
     topics = ("concurrency", "actor-framework", "actors", "agents", "sobjectizer")
-    settings = "compiler"
+    settings = "os", "compiler", "build_type", "arch"
```

Comment:
Why did you add these settings?

Reply:
Ohh brain fart https://github.com/conan-io/conan-center-index/pull/8670#issuecomment-1007397134